src/main/resources/static/BpdmCertificate.yml

```diff
@@ -0,0 +1,67 @@
+openapi: "3.0.2"
```

Comment:
Even to yaml files a copyright and license header should be added, as you did for `application.properties`. 

Please use .yaml as file extension, although .yml isn't wrong and also a well-known extension for YAML files.

Reply:
Used yaml as file extension